frappe/public/less/form.less

```diff
@@ -766,6 +766,7 @@ select.form-control {
 	color: @text-color;
 	font-size: 12px;
 	line-height: 1.7em;
+	margin-bottom: 10px !important;
```

Comment:
why? add margin-top on button

Reply:
Situtaion | On Preview             |  On Content
:-------------------------:|:-------------------------:|:-------------------------:
No CSS | ![screen shot 2018-11-02 at 12 39 58 pm](https://user-images.githubusercontent.com/8528887/47898615-a045e600-de9c-11e8-8624-36d30a759245.png)  |  ![screen shot 2018-11-02 at 12 40 58 pm](https://user-images.githubusercontent.com/8528887/47898626-a8058a80-de9c-11e8-80d4-50bcf72d33f2.png)
margin-top on button | ![screen shot 2018-11-02 at 12 43 35 pm](https://user-images.githubusercontent.com/8528887/47898712-016db980-de9d-11e8-97f0-3b1fdef88cda.png) | ![screen shot 2018-11-02 at 12 43 49 pm](https://user-images.githubusercontent.com/8528887/47898723-0894c780-de9d-11e8-9fbe-55eca2a70ff3.png)

Reply:
@netchampfaris What should I do here instead?